myapp/src/Day6/Que1/RouteComponent.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Suspense } from "react";
+import { Route, Link, Routes, BrowserRouter } from "react-router-dom";
+import { ErrorBoundary } from "react-error-boundary";
```

Comment:
@anantmishra-successive It is not compiling due to this. Again you have not tested. This is bad. plz remove.

Reply:
![resolveday6](https://github.com/anantmishra-successive/React_Training/assets/162098832/28277938-ac40-4358-a3b9-5249366e82d1)

Sir Now working , from next time without testing nothing will be pushed, In this case problem had arose due to not wrapping of routes under BrowserRouter. when first time i tested it worked , but during the resolving of conflicts it happened, from next time i will make sure to learn from this 

![testing](https://github.com/anantmishra-successive/React_Training/assets/162098832/a4c8b527-bbdc-4d8a-8f63-317db53acbf6)